cmd/metal-api/main.go

```diff
@@ -553,30 +558,31 @@ func initMasterData() {
 }
 
 func initIpam() {
-	dbAdapter := viper.GetString("ipam-db")
-	switch dbAdapter {
-	case "postgres":
-		pgStorage, err := goipam.NewPostgresStorage(
-			viper.GetString("ipam-db-addr"),
-			viper.GetString("ipam-db-port"),
-			viper.GetString("ipam-db-user"),
-			viper.GetString("ipam-db-password"),
-			viper.GetString("ipam-db-name"),
```

Comment:
Removed flags need to be removed from the cobra's root command as well. The new flag should be added to the root command.

Reply:
Done